module/mfrc522_user_command.c

```diff
@@ -86,6 +88,8 @@ static int mem_write(char *data)
 
 	pr_info("[MFRC522] Wrote data to memory\n");
 
+	stats->bytes_written += MFRC522_MEM_SIZE;
```

Comment:
```suggestion
	if (stats)
		stats->bytes_written += MFRC522_MEM_SIZE;
```
Look at review comment below